docs/user/openstack/install_upi.md

```diff
@@ -79,6 +76,34 @@ $ mkdir -p openstack-upi
 $ cd openstack-upi
 ```
 
+### Red Hat Enterprise Linux CoreOS (RHCOS)
+
+A proper RHCOS image in the OpenStack cluster or project is required for successful installation.
+
+Get the latest RHCOS image [here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/). You should download images with the highest version that is less than or equal to the OpenShift version that you install. Use the image versions that match your OpenShift version if they are available.
```

Comment:
This link points to 4.2 images. This won't work with 4.3.
Perhaps rephrase with `Get the RHCOS image for your OpenShift version here` pointing to https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/.

Reply:
4.3 images are not available at this moment. do you mean 4.1?